mentat/cost_tracker.py

```diff
@@ -44,6 +49,16 @@ def log_api_call_stats(
 
         costs_logger = logging.getLogger("costs")
         costs_logger.info(speed_and_cost_string)
+        self.last_api_call = speed_and_cost_string
+
+    def display_last_api_call(self):
```

Comment:
This isn't my favorite way to do this, but I think it's better than it was before. Before, whenever we called the llm api, we had to make a timer, end the timer, count the prompt and response tokens, and then call the log_api_call_stats function. Now, call_llm_api automatically logs it (but doesn't display it, since for a lot of things, like streaming changes, it would insert it in the middle of streaming), and stores the display string in last_api_call which can be displayed with this function. This means that the caller of call_llm_api can easily display the cost string whenever just by calling this function (although because of the way the response_logger_wrapper works, if we interrupt a streamed response it won't log, so we have to log manually in that case; but that only applied to one section of our codebase).